src/edu/ucsd/library/xdre/tab/SubjectTabularRecord.java

```diff
@@ -77,38 +79,47 @@ private void createSubject( Element e, Map<String,String> data, String ark ) thr
         	case "temporal":
         	case "topic":
                 // mats subject: elem, header, class/ns, predicate/ns, element, header exactMatch
-        		madsSubject( ark, e, subjectName, madsNS, subjectTerm, elemName, exactMatch );
+        		madsSubject( ark, e, subjectName, madsNS, subjectTerm, elemName, exactMatch, closeMatch );
```

Comment:
It looks like we're passing down this same set of, now, 8 parameters through a few functions. This makes me wonder if that data might warrant it's own small `Subject` or `SubjectNode` class, and then instances of that class could be passed as a single parameter between the functions. Just a thought, and I defer to whatever you think :) Otherwise, this all LGTM 👍

Reply:
It sounds good and I'll refactor it.